docs/screenshots/Makefile

```diff
@@ -0,0 +1,12 @@
+# Makefile
+# documents how to regenerate some .pngs
+#
+shoelaces-overview.png: shoelaces-overview.msc
+	msc-gen \
+		-T png \
+		-i shoelaces-overview.msc \
+		-o shoelaces-overview.png
+
+#
+# FYI  mscgen homepage is https://gitlab.com/msc-generator/msc-generator
+# l l
```

Comment:
Typo?

Reply:
`# l l`: a comment to express  _Last Line_.